examples/k8s/grafana-k8s/grafana-k8s.tf

```diff
@@ -0,0 +1,64 @@
+# This example deploys Grafana using Prometheus
+# as a data source.
+# Check https://charmhub.io/grafana-k8s for more details.
+#
+# After the deployment, get the admin password running:
+# `juju run-action grafana/0 get-admin-password --wait`
+#
+# Next, browse to the main Grafana login page. Use the 
+# grafana/0 unit IP and port 3000 (e.g. http://10.1.250.216:3000)
```

Comment:
Should we not be using ingress so that we don't have to rely on hitting the unit/IP of the grafana instance directly? This will only work if you're using MicroK8s but not on other Kubernetes clusters I believe (due to MicroK8s making those IPs accessible from the host machine).